src/mpid/ch4/subconfigure.m4

```diff
@@ -249,14 +249,14 @@ AC_ARG_ENABLE(ch4-direct,
 AC_ARG_WITH(ch4-shmmods,
     [  --with-ch4-shmmods@<:@=ARG@:>@ Specify the shared memory submodules for MPICH/CH4.
                           Valid options are:
-                          posix_only   - Only enable POSIX SHM (default)
-                          xpmem        - Enable XPMEM SHM for partial communication paths and use
-                                         POSIX SHM as fallback for others
-                          gpu          - Enable GPU SHM for partial communication paths and use
-                                         POSIX SHM as fallback for others
+                          auto         - Enable everything that is available/allowed by netmod (default)
+                                         (cannot be combined with other options)
+                          posix        - Enable POSIX shmmod
+                          xpmem        - Enable XPMEM IPC (requires posix)
+                          gpu          - Enable GPU Direct IPC (requires posix)
                  ],
```

Comment:
I think we need to mention that the value should be either a `comma-separated-list` or `auto`.

Reply:
Fixed.